src/trading-api/next/v1/bullish-trading-api.yml

```diff
@@ -3077,27 +3077,56 @@ components:
     TradingAccountResponse:
       type: object
       required:
-        - tradingAccountId
-        - tradingAccountName
-        - tradingAccountDescription
+        - isBorrowing
+        - isLending
         - isPrimaryAccount
+        - makerFee
+        - maxInitialLeverage
         - rateLimitToken
+        - takerFee
+        - tradingAccountDescription
+        - tradingAccountId
+        - tradingAccountName
       properties:
+        isBorrowing:
+          description: Whether the trading account is borrowing a loan
+          type: boolean
+          example: false
+        isLending:
+          description: Whether the trading account is lending a loan
```

Comment:
Whether the trading account has opened loan offers

Reply:
I just updated it to `whether the trading account is lending` just to keep it consistent withe definition of `isBorrowing` (whether the trading_account is borrowing)